docker-compose.yaml

```diff
@@ -3,7 +3,7 @@
 
 services:
   dev-portal:
-    image: 'hashicorp/dev-portal:latest'
+    image: 'dev-portal:waf-docs'
```

Comment:
@im2nguyen what is the difference between these two images of dev-portal?

Reply:
It's updating it from tutorials to docs: https://github.com/hashicorp/dev-portal/pull/2763